it/uk/gov/hmrc/minorentityidentificationfrontend/views/trustViews/CannotConfirmBusinessViewTests.scala

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.minorentityidentificationfrontend.views.trustViews
+
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import play.api.libs.ws.WSResponse
+import uk.gov.hmrc.minorentityidentificationfrontend.assets.MessageLookup.{Base, BetaBanner, Header, CannotConfirmBusiness => messages}
+import uk.gov.hmrc.minorentityidentificationfrontend.assets.TestConstants.testSignOutUrl
+import uk.gov.hmrc.minorentityidentificationfrontend.config.AppConfig
+import uk.gov.hmrc.minorentityidentificationfrontend.utils.ComponentSpecHelper
+import uk.gov.hmrc.minorentityidentificationfrontend.utils.ViewSpecHelper.ElementExtensions
+
+
+trait CannotConfirmBusinessViewTests {
+  this: ComponentSpecHelper =>
+
+  def testCannotConfirmBusinessView(result: => WSResponse): Unit = {
+    lazy val doc: Document = Jsoup.parse(result.body)
+    lazy val config = app.injector.instanceOf[AppConfig]
+
+    "have a sign out link in the header" in {
+      doc.getSignOutText mustBe Header.signOut
+    }
+
+    "have sign out link redirecting to signOutUrl from journey config" in {
+      doc.getSignOutLink mustBe testSignOutUrl
+    }
+
+    "have the correct beta banner" in {
+      doc.getBanner.text mustBe BetaBanner.title
+      doc.getElementsByClass("govuk-link").get(1).attr("href") mustBe config.betaFeedbackUrl("vrs")
+    }
+
+    "have the correct title" in {
+      doc.title mustBe messages.title
+    }
+
+    "have the correct heading" in {
+      doc.getH1Elements.get(0).text mustBe messages.heading
+    }
+
+    "have the correct first line" in {
+      doc.getParagraphs.get(1).text mustBe messages.line_1
+    }
+
+    "have correct text on the radio buttons" in {
+      doc.getLabelElement.first.text() mustBe Base.yes
+      doc.getLabelElement.get(1).text() mustBe Base.no
+      doc.getH1Elements.get(1).text() mustBe messages.radio
```

Comment:
Radio isn't the most intuitive of names I'd tend to use "question".

Reply:
Updated